### PR TITLE
Replace object type with verbose name

### PR DESCRIPTION
**Pull request by DIT-CJWaszczuk**
## Summary
Apart from Quotas (we use `QuotaOrderNumber` for the data), we can remove the need for object_type and replace it with the verbose names.

### CODE REVIEW

File: common/jinja2/layouts/detail.jinja [hunk 1]
```diff
@@ -1,12 +1,10 @@
 {% extends "layouts/layout.jinja" %}
 
-{% set object_type_plural = object_type_plural or object_type + "s" %}
-
 {% block breadcrumb %}
   {{ govukBreadcrumbs({
     "items": [
       {"text": "Home", "href": url("index")},
-      {"text": "Find and edit " ~ object_type_plural, "href": find_edit_url},
+      {"text": "Find and edit " ~ object._meta.verbose_name_plural, "href": find_edit_url},
```
Comment (andyhd) [inline common/jinja2/layouts/detail.jinja:7]: I have a bit in my PR that allows you to do `object.get_url(action="list")` so you can get rid of `find_edit_url` too

**Review by andyhd — APPROVED** [review]
Looks good to me

**Review by stuaxo — APPROVED** [review]
Nice, there's so much handy stuff in _meta.

Good how this removes a bunch of variables.

**Comment by DIT-CJWaszczuk** [comment]
This actually would not work, not in its current state anyway. If there are no results, there's no object to get the verbose name  from. Hmmm

File: common/jinja2/layouts/list.jinja [hunk 2]
```diff
@@ -4,10 +4,10 @@
 {% from "components/input/macro.njk" import govukInput %}
 {% from "components/table/macro.njk" import govukTable %}
 
-{% set object_type_plural = object_type_plural or object_type + "s" %}
 {% set create_url = create_url or "create" %}
+{% set object = object_list|first %}
 
-{% set page_title = "Find and edit " + (object_type_plural or object_type + "s") %}
+{% set page_title = "Find and edit " ~ object._meta.verbose_name_plural %}
```
Comment (andyhd) [inline common/jinja2/layouts/list.jinja:10]: This needs to be `object_list.model._meta.verbose_name_plural`

**Review by andyhd — CHANGES_REQUESTED** [review]

**Review by simonwo — APPROVED** [review]

**Pull request merged**